website/content/docs/discovery/dns.mdx

```diff
@@ -136,17 +136,22 @@ it is recommended to use the HTTP API to retrieve the list of nodes.
 The format of a standard service lookup is:
 
 ```text
-[<tag>.]<service>.service[.<datacenter>].<domain>
+[<tag>.]<service>.service[.<datacenter-or-peer>].<domain>
 ```
 
-The `tag` is optional, and, as with node lookups, the `datacenter` is as
-well. If no tag is provided, no filtering is done on tag. If no
-datacenter is provided, the datacenter of this Consul agent is assumed.
+The `tag` is optional. If no tag is provided, no filtering is done on tag.
+
+The `datacenter-or-peer` is also optional.
+If `datacenter-or-peer` is not provided, the datacenter of the Consul agent is assumed.
+A provided value must be the name a WAN federated datacenter or a cluster peer
+of the Consul agent acting as the DNS server.
+To lookup a mesh-participating service that's imported from a cluster peer,
+refer to [service virtual IP lookup](#service-virtual-ip-lookups) instead.
 
 If we want to find any redis service providers in our local datacenter,
 we could query `redis.service.consul.` If we want to find the PostgreSQL
-primary in a particular datacenter, we could query
-`primary.postgresql.service.dc2.consul.`
+primary in a WAN federated datacenter or cluster peer named `dc2`,
+we could query `primary.postgresql.service.dc2.consul.`
```

Comment:
Intended to support in the future, but no support now since it's always parsed as dc